nrf_modem/doc/dectphy.rst

```diff
@@ -0,0 +1,302 @@
+.. _nrf_modem_dect_phy:
+
+DECT NR+ physical layer
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The DECT physical layer (PHY) interface in the Modem library is used to control the DECT NR+ PHY in the DECT NR+ PHY firmware.
+
+The DECT NR+ PHY firmware is a variant of the nRF9161 firmware with a different feature set than the cellular firmware.
```

Comment:
Should we refer as nRF91x1 modem firmware, considering this will be also applicable for nRF9151 as well?